mongospecs/base.py

```
@@ -1,852 +1,26 @@
-
-from .query import Condition, Group
-
-FilterType = t.Union[None, t.MutableMapping[str, t.Any], Condition, Group]
-SpecDocumentType = t.TypeVar("SpecDocumentType", bound=t.MutableMapping[str, t.Any], covariant=True)
-Specs = t.Sequence["SpecBase[SpecDocumentType]"]
-RawDocuments = t.Sequence[SpecDocumentType]
-SpecsOrRawDocuments = t.Sequence[t.Union["SpecBase[SpecDocumentType]", SpecDocumentType]]
+from .mixins.crud import CrudMixin
+from .mixins.index import IndexManagementMixin
+from .mixins.integrity import IntegrityMixin
+from .mixins.query import QueryMixin
```

Ruff (0.8.2)

`.mixins.query.QueryMixin` imported but unused

Remove unused import: `.mixins.query.QueryMixin`

(F401)